### PR TITLE
docs: add more examples

### PR DESCRIPTION
**Pull request by martinszeltins**
I added more examples to the docs. These are things that I myself have found very useful in the past.

Currently these examples are hosted on my own GitHub, maybe you want to move them to "regle" repo or something.

### CODE REVIEW

File: docs/.vitepress/theme/style.css [hunk 1]
```diff
@@ -128,3 +128,16 @@
 .DocSearch {
   --docsearch-primary-color: var(--vp-c-brand-1) !important;
 }
+
+/**
+ * Give example pages more space to show the Stackblitz iframe embed,
+ * we don't need the TOC there anyway.
+ * -------------------------------------------------------------------------- */
+
+.container:has(.vp-doc[class*="_examples_"]) > div.aside {
```
Comment (victorgarciaesgi) [inline docs/.vitepress/theme/style.css:137]: smart!

Comment (martinszeltins) [reply docs/.vitepress/theme/style.css:137]: hehe 😄

**Comment by victorgarciaesgi** [comment]
Don't worry it was my fault the build was broken

**Comment by victorgarciaesgi** [comment]
> Currently these examples are hosted on my own GitHub, maybe you want to move them to "regle" repo or something.

Yeah I think it can become hard to maintain if we have multiple repos at different places, i'll see if we can regroup it all in one place

**Comment by victorgarciaesgi** [comment]
I've made a central repo for all examples!
https://github.com/victorgarciaesgi/regle-examples

**Comment by martinszeltins** [comment]
> I've made a central repo for all examples! [victorgarciaesgi/regle-examples](https://github.com/victorgarciaesgi/regle-examples)

👍🏻

**Comment by victorgarciaesgi** [comment]
I modified some urls in the main branch

- iframe src would be `https://stackblitz.com/github/victorgarciaesgi/regle-examples/tree/main/examples/XXX-example?file=src%2FApp.vue&theme=dark&view=preview`
- `Open in stackblitz` button href would be `https://stackblitz.com/~/github.com/victorgarciaesgi/regle-examples/tree/main/examples/XXX-example?file=examples/XXX-example/src/App.vue&configPath=examples/XXX-example`

**Comment by victorgarciaesgi** [comment]
Don't forget to add them in the `regle-examples/pnpm-workspace.yaml`

**Comment by martinszeltins** [comment]
Moved all examples to `regle-examples` repo

**Comment by martinszeltins** [comment]
Moved all examples to https://github.com/victorgarciaesgi/regle-examples/pull/2

**Pull request merged**